### ISSUE TITLE
Strategy for creating ground truth 

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/pipeline-ms-lesion#2]
A straightforward approach to set up our training dataset is co-register all contrasts, so that we only need to perform one labeling per patient. 

In order to always work in a fixed resolution, we could maybe consider straightening all contrasts, and interpolating them to a 0.5mm iso resolution. This target resolution is a reasonable tradeoff between required resolution (for lesion segmentation) and computation time.

Pros/Cons of labeling in common space:
- We only deal with one label per subject (instead of one per contrast per subject).
- We benefit from having the other contrasts registered (easier to spot artifacts).

Pros/Cons of labeling in native space:
- Pros
  - there is redundancy on the GT (more reliable wrt. intra-rater variability)
  - labeling takes less time because resolution is coarser (e.g. 0.5x0.5x3mm vs. 0.5mm iso)
  - we could combine the GT from each contrast using STAPLE_soft, average, etc.
  - easier to correct a GT starting from the native image (because processing the image might be complicated in some cases, e.g multi-center studies, etc.)
- Cons:
  - If we have many contrasts (eg 5), labeling needs to be done for each of them

3rd option (hybrid):
- perform GT in common space and then apply inverse warp to get it in native space.

**Comment by jcohenadad** [comment]
After discussing internally, we decided to do the ground truth segmentations on the native images. Arguments are:
- It takes less time to perform the segmentation on the native resolution (e.g. 0.5x0.5x3) compared to 0.5mm iso.
- Having the segmentation for each contrast (T2, T2*, etc.) would make it possible to then combine each segmentation and thus improve the precision (as opposed to having only one segmentation per subject).
- The co-registration procedure might improve in the future, therefore the segmentations we create now will not be bound to the image registration. 

Next steps:
- Set up a procedure for centralizing the manual segmentations
- Set up a procedure for describing the manual segmentation procedure. We could start from the instructions given for the spinal MS lesion mapping project:

  ![image](https://user-images.githubusercontent.com/2482071/104033145-248f0480-519d-11eb-9e99-be0966749244.png)
 
- Things to add to this procedure: 
  - Generation of a JSON file
  - Create a script that loops across subjects, open the image editor and generate the JSON file. Similar to what is done for the [spine generic project](https://github.com/spine-generic/spine-generic/blob/master/spinegeneric/cli/manual_correction.py)

**Comment by charleygros** [comment]
Email from rater # 1 (20/11/2020) says: first time point only and on T2star only.
